bin/get-hetzner-cloud-token

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+set -eux
+
+curl \
+  -A 'travis-ansible-provider' \
```

Comment:
@LKaemmerling, is this OK? I switched it from "travis-terraform-provider" ...

Reply:
Yep is okay :)

Reply:
Thanks!